### PR TITLE
(Re?)adds move_force

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request
We had no way to give push priority (Which mechs need) because for some reason we only have a half-implemented move_resist

Idk why the safe throw was commented out it's only used for shuttles so whats the worst

## Changelog
:cl:
code: Added push priorities
/:cl:

### CODE REVIEW

**Review by BraveMole — CHANGES_REQUESTED** [review]
I do not want to have two push/no push systems, make that consistent with xeno/marine push please

File: code/game/atoms_movable.dm [hunk 1]
```diff
@@ -1040,6 +1042,25 @@
 		return
 	DISABLE_BITFIELD(flags_pass, HOVERING)
 
+///returns bool for if we want to get forcepushed
+/atom/movable/proc/force_pushed(atom/movable/pusher, force = MOVE_FORCE_DEFAULT, direction)
```
Comment (BraveMole) [inline code/game/atoms_movable.dm:1046]: Why is this a proc

File: code/game/atoms_movable.dm [hunk 2]
```diff
@@ -1040,6 +1042,25 @@
...
+		visible_message(span_warning("[src] forcefully pushes against [pushed_atom]!"), span_warning("You forcefully push against [pushed_atom]!"))
+
+///returns bool for if we want to get handle move crushing, return is bool if we can move an anchored obj
+/atom/movable/proc/move_crush(atom/movable/crushed_atom, force = move_force, direction, silent = FALSE)
+	. = crushed_atom.move_crushed(src, force, direction)
+	if(!silent && .)
+		visible_message(span_danger("[src] crushes past [crushed_atom]!"), span_danger("You crush [crushed_atom]!"))
+
+///returns bool for if we want to get crushed
+/atom/movable/proc/move_crushed(atom/movable/pusher, force = MOVE_FORCE_DEFAULT, direction)
```
Comment (BraveMole) [inline code/game/atoms_movable.dm:1062]: Ditto

**Review by BraveMole — CHANGES_REQUESTED** [review]

File: code/game/atoms_movable.dm (hunk 1 above)
Comment (TiviPlus) [reply code/game/atoms_movable.dm:1046]: TG port its so damage and etc can be handled as needed

**Pull request merged**